ffi/wgpu.h

```diff
@@ -161,21 +161,23 @@ WGPUPresentMode const * wgpuSurfaceGetSupportedPresentModes(WGPUSurface surface,
 
 void wgpuRenderPassEncoderSetPushConstants(WGPURenderPassEncoder encoder, WGPUShaderStageFlags stages, uint32_t offset, uint32_t sizeBytes, void* const data);
 
+void wgpuAdapterDrop(WGPUAdapter adapter);
```

Comment:
You just rearranged these to be in order?

Reply:
yep, and added missing definitions for `wgpuAdapterDrop` & `wgpuSurfaceDrop`